public/locales/en-US/grafana.json

```diff
@@ -10312,7 +10312,7 @@
     },
     "invite-user": {
       "invite-button": "Invite",
-      "invite-new-member-button": "Invite new member",
+      "invite-new-user-button": "Invite new user",
```

Comment:
In the beginning, we used member because it was the word used in the pages `/admin/users` and `grafana.com`. Now I see we are using `users` in `/admin/users`, so I think this change makes sense. However, we are still using `members` in `grafana.com`.
This is probably due to a member of Grafana.com representing more than a Grafana user. I don't know how to solve this inconsistency but I wanted to raise this concern so we are all aware of it.